src/examples/rest/src/backend/schema/tag/entity.ts

```diff
@@ -29,7 +29,7 @@ const acl: AccessControlList<Tag, AuthorizationContext> = {
 };
 
 @ApplyMultiFactorAuthentication<Tag>(() => ({
-	Everyone: {
+	LIGHT_SIDE: {
```

Comment:
In order to test delete I have changed this MFA to only trigger for Light Side role.